dxp/architecture/rich_text_editors.md

```diff
@@ -14,11 +14,11 @@ If you need to use a WYSIWYG editor in your application, please identify the dif
 
 Depending on the use case, your application needs likely fall in one of these categories
 
-| Rich Text Needs | Solution |
-| --- | --- |
-| No need for rich text features (bold, links...) | Use simple input elements like **`<aui:input />`** or **`<input />`** |
-| Basic Rich Text Features (text formatting, links, images...) | Use **`<liferay-ui:input-editor />`** or the **`<Editor />`** React component
-| Contextual Inline Editing Experience (page-editing, blogs...) | Use **`<liferay-ui:input-editor type="balloon" />`** or the **`<BallonEditor />`** React component |
+| Rich Text Needs                                               | Solution                                                                                           |
+| ------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| No need for rich text features (bold, links...)               | Use simple input elements like **`<aui:input />`** or **`<input />`**                              |
+| Basic Rich Text Features (text formatting, links, images...)  | Use **`<liferay-ui:input-editor />`** or the **`<Editor />`** React component                      |
```

Comment:
Should `<liferay-ui:input-editor>` be `<liferay-editor:editor>` ?

Reply:
Nevermind, I saw the warning below (on line 47)

Reply:
Good catch though. Maybe this could be

> Use **`<liferay-ui:input-editor />`** (deprecated) or ...